modules/KalturaSupport/components/share/share.js

```diff
@@ -172,7 +172,42 @@
 					_this.getScreen().then(function(screen){
 						$( "#" + embedPlayer.getPlayerElement().id ).addClass("blur");
 						embedPlayer.getPlayerPoster().addClass("blur");
+						embedPlayer.getInterface().find(".share .icon-close").focus();
```

Comment:
why not use getScreen?

Reply:
@MaksymKorniiov  can you please review this request?